handout/directives/ng_for_directive.md

```diff
@@ -1,6 +1,6 @@
 # NgFor Directive
 
-The `ngFor` directive is a way of repeating a template by using each item of an iterable as that template's context.
+The `ngFor` directive is a way of repeatly adding content inside the ngFor block to the DOM.
```

Comment:
We'll it doesn't repeat the entire template, this is incorrect, only the content inside the ngFor is repeated. If you have HTML and other angular "template" code outside ngFor score, that will not be repeated. Here I believe there is an error in the Angular 2 doc which use templates and not content. I think my change makes more sense, we ask an additional reviewer to comment?